fe/fe-core/src/main/java/com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java

```diff
@@ -319,6 +319,7 @@ private void planMVQuery(CreateMaterializedViewStatement createStmt, QueryStatem
                 ExecPlan execPlan =
                         PlanFragmentBuilder.createPhysicalPlanForMV(ctx, createStmt, optimizedPlan, logicalPlan,
                                 queryRelation, columnRefFactory);
+                createStmt.setMaintenancePlan(execPlan, columnRefFactory);
```

Comment:
It has been set in `PlanFragmentBuilder::createPhysicalPlanForMV`

Reply:
fixed.